### DISCUSSION TITLE
In the colab notebook for finetuning, 1. are we doing finetuning all the parameterts in llm 2. are we performing lora techniques

### DISCUSSION THREAD

**Discussion by kishoreb4** [time-series-foundation-models/lag-llama#101] [Q&A] [answered]
I ran the colab notebook for finetuning, Just want to know what it comes under.

**Comment by ashok-arjun** [comment, marked as answer]
We are doing (1)

**Reply by kishoreb4** [reply]
Thanks @ashok-arjun for you reply.
Also, one more question. I have fine tuned 3000 items weekly units sold. and results are not that great. Do we need to standardize or preprocess data in any form before finetuning?
In the colab demo file the dataset has 10 items and target values seem to have mean close 0 and std close to 1.
Waiting for you reply.
